### PR TITLE
add strdst token

### PR DESCRIPTION
**Pull request by lakmalmabm**
Adding token: STRDST from chain Stargaze

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -1569,6 +1569,16 @@
       "chain_name": "osmosis",
       "base_denom": "factory/osmo1s794h9rxggytja3a4pmwul53u98k06zy2qtrdvjnfuxruh7s8yjs6cyxgd/umbrn",
       "osmosis_verified": true
+    },
+    {
+      "chain_name": "stargaze",
+      "base_denom": "factory/stars16da2uus9zrsy83h23ur42v3lglg5rmyrpqnju4/dust",
+      "path": "transfer/channel-75/factory/stars16da2uus9zrsy83h23ur42v3lglg5rmyrpqnju4/dust",
+      "osmosis_main": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:1577]: please replace osmosis_main and osmosis_frontier with osmosis_verified, and please default to false.

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -1569,6 +1569,16 @@
...
+      "osmosis_frontier": true,
+      "pools": {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:1579]: We've stopped using the 'pools' property in this way, but it's good to ensure the description contains the pool id as that help us set up.

**Comment by JeremyParish69** [comment]
apparently pool 1234

**Comment by lakmalmabm** [comment]
Hi @JeremyParish69  is there anything need from my side?

**Comment by JeremyParish69** [comment]
> Hi @JeremyParish69 is there anything need from my side?

Just building a preview frontend for validation now. If everything's working then it should be listed within a couple days

**Comment by lakmalmabm** [comment]
sure @JeremyParish69  thank you very much.

**Comment by lakmalmabm** [comment]
@JeremyParish69 will it take more days to show the details in the frontend after the PR is merged?

https://app.osmosis.zone/pool/1234

I cannot see the details yet. still does not show the name and logo.

**Pull request merged**